src/AuditableObserver.php

```diff
@@ -40,7 +40,16 @@ public function created(AuditableContract $model)
      */
     public function updated(AuditableContract $model)
     {
-        Auditor::execute($model->setAuditEvent('updated'));
+        if (
+            method_exists($model, 'getDeletedAtColumn') &&
+            $model->isDirty($model->getDeletedAtColumn()) &&
+            !$model->deleted_at &&
+            count($model->getDirty()) == 2 //deleted_at and updated_at
```

Comment:
While this may work, I don't like how the code is written. Specially the empty `if` part.